### PR TITLE
Qtile wayland

### LINKED ISSUE #45429: Package request: python-pywlroots
### Package name

python-pywlroots

### Package homepage

https://github.com/flacjacket/pywlroots

### Description

Python binding to the wlroots library using cffi as required by Qtile if you're running it with the Wayland backend

### Does the requested package meet the package requirements?

Required

### Is the requested package released?

Yes

### LINKED ISSUE #45430: Package request: python-pywayland
### Package name

python-pywayland

### Package homepage

https://github.com/flacjacket/pywayland

### Description

PyWayland provides a wrapper to the libwayland library using the CFFI library to provide access to the Wayland library calls and written in pure Python and is required if you're running Qtile with the Wayland backend.

### Does the requested package meet the package requirements?

Required

### Is the requested package released?

Yes

### LINKED ISSUE #45431: Package request: python-xkbcommon
### Package name

python-xkbcommon

### Package homepage

https://github.com/sde1000/python-xkbcommon

### Description

Python bindings for libxkbcommon using cffi required if you're running Qtile with the Wayland backend.

### Does the requested package meet the package requirements?

Required

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by sgn**
Close #45429 
Close #45430 
Close #45431 

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/qtile/template [hunk 1]
```diff
@@ -18,3 +20,11 @@ post_install() {
 	vinstall resources/qtile.desktop 644 usr/share/xsessions
 	vlicense LICENSE
 }
+
+qtile-wayland_package() {
+	depends="${sourcepkg}>=${version}_${revision}
+	 python3-pywlroots-${_wlroots} python3-pywayland python3-xkbcommon"
```
Comment (classabbyamp) [inline srcpkgs/qtile/template:26]: are pywaland and xkbcommon necessary to list here? they're already dependencies of pywlroots

File: srcpkgs/python3-pywlroots-0.15/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pywlroots-0.15'
+pkgname=python3-pywlroots-0.15
```
Comment (mhmdanas) [inline srcpkgs/python3-pywlroots-0.15/template:2]: I'd name it `python3-pywlroots0.15` instead, as it's more in line with how other versioned packages are named.

File: srcpkgs/python3-pywlroots-0.15/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+version=0.15.24
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-cffi python3-pywayland python3-xkbcommon
+ python3-wheel"
+makedepends="python3-devel wlroots${pkgname##*-}-devel"
+depends="python3-pywayland python3-xkbcommon python3-cffi"
+short_desc="Python binding to the wlroots ${pkgname##*-} using cffi"
```
Comment (mhmdanas) [inline srcpkgs/python3-pywlroots-0.15/template:10]:
```suggestion
short_desc="Python bindings to wlroots ${pkgname##*-} using cffi"
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-pywlroots-0.15/template (hunk 3 above)
Comment (sgn) [reply srcpkgs/python3-pywlroots-0.15/template:10]: See https://pypi.org/project/pywlroots/

File: srcpkgs/python3-pywlroots-0.15/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/python3-pywlroots-0.15/template:2]: This naming schema is easier to manipulate, though.

**Comment by sgn** [comment]
Gonna merge this later today if noone adds objections.

**Pull request merged**